habinezadalvan/haven
# Ft signup ui

**Description of Task to be completed?**

- created sign up and login pages with HTML and sass

**How should this be manually tested?**

- clone the repo
-make sure you have Live Sass Compiler in your editor
- go the folder directory 
- go in your terminal watch sass to compile into CSS
- open signup.html and login.html in your browser

**Any background context you want to provide?**

- used sass, HTML

**What are the relevant pivotal tracker stories?**
no story created

Screenshots (if appropriate)
![1-12902_facebook-logo-png-white-facebook-logo-png-white](https://user-images.githubusercontent.com/45101716/64925635-fa6d4280-d7f3-11e9-842f-7b4a8f381eec.png)
![Screen Shot 2019-09-15 at 20 03 39](https://user-images.githubusercontent.com/45101716/64925636-fccf9c80-d7f3-11e9-9952-e0cb37b7832c.png)
